[2020-02-10 18:14:39] console.ERROR: Error thrown while running command "make:migration". Message: "An exception occurred in driver: SQLSTATE[HY000] [2002] Connection refused" {"exception":"[object] (Doctrine\\DBAL\\Exception\\ConnectionException(code: 0): An exception occurred in driver: SQLSTATE[HY000] [2002] Connection refused at /Applications/MAMP/htdocs/recettes/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/AbstractMySQLDriver.php:93, Doctrine\\DBAL\\Driver\\PDOException(code: 2002): SQLSTATE[HY000] [2002] Connection refused at /Applications/MAMP/htdocs/recettes/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php:31, PDOException(code: 2002): SQLSTATE[HY000] [2002] Connection refused at /Applications/MAMP/htdocs/recettes/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php:27)","command":"make:migration","message":"An exception occurred in driver: SQLSTATE[HY000] [2002] Connection refused"} []
[2020-02-10 18:14:39] console.DEBUG: Command "make:migration" exited with code "1" {"command":"make:migration","code":1} []
[2020-02-10 18:17:41] console.ERROR: Error thrown while running command "make:migration". Message: "An exception occurred in driver: SQLSTATE[HY000] [2002] Connection refused" {"exception":"[object] (Doctrine\\DBAL\\Exception\\ConnectionException(code: 0): An exception occurred in driver: SQLSTATE[HY000] [2002] Connection refused at /Applications/MAMP/htdocs/recettes/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/AbstractMySQLDriver.php:93, Doctrine\\DBAL\\Driver\\PDOException(code: 2002): SQLSTATE[HY000] [2002] Connection refused at /Applications/MAMP/htdocs/recettes/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php:31, PDOException(code: 2002): SQLSTATE[HY000] [2002] Connection refused at /Applications/MAMP/htdocs/recettes/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php:27)","command":"make:migration","message":"An exception occurred in driver: SQLSTATE[HY000] [2002] Connection refused"} []
[2020-02-10 18:17:41] console.DEBUG: Command "make:migration" exited with code "1" {"command":"make:migration","code":1} []
[2020-02-10 18:18:09] console.ERROR: Error thrown while running command "make:migration". Message: "An exception occurred in driver: SQLSTATE[HY000] [2002] No such file or directory" {"exception":"[object] (Doctrine\\DBAL\\Exception\\ConnectionException(code: 0): An exception occurred in driver: SQLSTATE[HY000] [2002] No such file or directory at /Applications/MAMP/htdocs/recettes/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/AbstractMySQLDriver.php:93, Doctrine\\DBAL\\Driver\\PDOException(code: 2002): SQLSTATE[HY000] [2002] No such file or directory at /Applications/MAMP/htdocs/recettes/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php:31, PDOException(code: 2002): SQLSTATE[HY000] [2002] No such file or directory at /Applications/MAMP/htdocs/recettes/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php:27)","command":"make:migration","message":"An exception occurred in driver: SQLSTATE[HY000] [2002] No such file or directory"} []
[2020-02-10 18:18:09] console.DEBUG: Command "make:migration" exited with code "1" {"command":"make:migration","code":1} []
[2020-02-10 18:18:30] console.ERROR: Error thrown while running command "make:migration". Message: "An exception occurred in driver: SQLSTATE[HY000] [2002] No such file or directory" {"exception":"[object] (Doctrine\\DBAL\\Exception\\ConnectionException(code: 0): An exception occurred in driver: SQLSTATE[HY000] [2002] No such file or directory at /Applications/MAMP/htdocs/recettes/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/AbstractMySQLDriver.php:93, Doctrine\\DBAL\\Driver\\PDOException(code: 2002): SQLSTATE[HY000] [2002] No such file or directory at /Applications/MAMP/htdocs/recettes/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php:31, PDOException(code: 2002): SQLSTATE[HY000] [2002] No such file or directory at /Applications/MAMP/htdocs/recettes/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php:27)","command":"make:migration","message":"An exception occurred in driver: SQLSTATE[HY000] [2002] No such file or directory"} []
[2020-02-10 18:18:30] console.DEBUG: Command "make:migration" exited with code "1" {"command":"make:migration","code":1} []
[2020-02-10 18:30:55] console.ERROR: Error thrown while running command "make:migration". Message: "An exception occurred in driver: SQLSTATE[HY000] [2002] No such file or directory" {"exception":"[object] (Doctrine\\DBAL\\Exception\\ConnectionException(code: 0): An exception occurred in driver: SQLSTATE[HY000] [2002] No such file or directory at /Applications/MAMP/htdocs/recettes/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/AbstractMySQLDriver.php:93, Doctrine\\DBAL\\Driver\\PDOException(code: 2002): SQLSTATE[HY000] [2002] No such file or directory at /Applications/MAMP/htdocs/recettes/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php:31, PDOException(code: 2002): SQLSTATE[HY000] [2002] No such file or directory at /Applications/MAMP/htdocs/recettes/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php:27)","command":"make:migration","message":"An exception occurred in driver: SQLSTATE[HY000] [2002] No such file or directory"} []
[2020-02-10 18:30:55] console.DEBUG: Command "make:migration" exited with code "1" {"command":"make:migration","code":1} []
[2020-02-10 18:31:08] console.ERROR: Error thrown while running command "make:migration". Message: "An exception occurred in driver: SQLSTATE[HY000] [2002] No such file or directory" {"exception":"[object] (Doctrine\\DBAL\\Exception\\ConnectionException(code: 0): An exception occurred in driver: SQLSTATE[HY000] [2002] No such file or directory at /Applications/MAMP/htdocs/recettes/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/AbstractMySQLDriver.php:93, Doctrine\\DBAL\\Driver\\PDOException(code: 2002): SQLSTATE[HY000] [2002] No such file or directory at /Applications/MAMP/htdocs/recettes/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php:31, PDOException(code: 2002): SQLSTATE[HY000] [2002] No such file or directory at /Applications/MAMP/htdocs/recettes/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php:27)","command":"make:migration","message":"An exception occurred in driver: SQLSTATE[HY000] [2002] No such file or directory"} []
[2020-02-10 18:31:08] console.DEBUG: Command "make:migration" exited with code "1" {"command":"make:migration","code":1} []
[2020-02-10 18:31:23] console.ERROR: Error thrown while running command "make:migration". Message: "An exception occurred in driver: SQLSTATE[HY000] [2002] Connection refused" {"exception":"[object] (Doctrine\\DBAL\\Exception\\ConnectionException(code: 0): An exception occurred in driver: SQLSTATE[HY000] [2002] Connection refused at /Applications/MAMP/htdocs/recettes/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/AbstractMySQLDriver.php:93, Doctrine\\DBAL\\Driver\\PDOException(code: 2002): SQLSTATE[HY000] [2002] Connection refused at /Applications/MAMP/htdocs/recettes/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php:31, PDOException(code: 2002): SQLSTATE[HY000] [2002] Connection refused at /Applications/MAMP/htdocs/recettes/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php:27)","command":"make:migration","message":"An exception occurred in driver: SQLSTATE[HY000] [2002] Connection refused"} []
[2020-02-10 18:31:23] console.DEBUG: Command "make:migration" exited with code "1" {"command":"make:migration","code":1} []
[2020-02-10 18:34:58] console.ERROR: Error thrown while running command "'d:s:u'". Message: "An exception occurred in driver: SQLSTATE[HY000] [2002] Connection refused" {"exception":"[object] (Doctrine\\DBAL\\Exception\\ConnectionException(code: 0): An exception occurred in driver: SQLSTATE[HY000] [2002] Connection refused at /Applications/MAMP/htdocs/recettes/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/AbstractMySQLDriver.php:93, Doctrine\\DBAL\\Driver\\PDOException(code: 2002): SQLSTATE[HY000] [2002] Connection refused at /Applications/MAMP/htdocs/recettes/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php:31, PDOException(code: 2002): SQLSTATE[HY000] [2002] Connection refused at /Applications/MAMP/htdocs/recettes/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php:27)","command":"'d:s:u'","message":"An exception occurred in driver: SQLSTATE[HY000] [2002] Connection refused"} []
[2020-02-10 18:34:58] console.DEBUG: Command "'d:s:u'" exited with code "1" {"command":"'d:s:u'","code":1} []
[2020-02-10 18:37:26] console.ERROR: Error thrown while running command "'d:s:u'". Message: "An exception occurred in driver: SQLSTATE[HY000] [2002] Connection refused" {"exception":"[object] (Doctrine\\DBAL\\Exception\\ConnectionException(code: 0): An exception occurred in driver: SQLSTATE[HY000] [2002] Connection refused at /Applications/MAMP/htdocs/recettes/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/AbstractMySQLDriver.php:93, Doctrine\\DBAL\\Driver\\PDOException(code: 2002): SQLSTATE[HY000] [2002] Connection refused at /Applications/MAMP/htdocs/recettes/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php:31, PDOException(code: 2002): SQLSTATE[HY000] [2002] Connection refused at /Applications/MAMP/htdocs/recettes/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php:27)","command":"'d:s:u'","message":"An exception occurred in driver: SQLSTATE[HY000] [2002] Connection refused"} []
[2020-02-10 18:37:26] console.DEBUG: Command "'d:s:u'" exited with code "1" {"command":"'d:s:u'","code":1} []
[2020-02-10 18:39:11] console.ERROR: Error thrown while running command "'d:s:u'". Message: "An exception occurred in driver: SQLSTATE[HY000] [2002] Connection refused" {"exception":"[object] (Doctrine\\DBAL\\Exception\\ConnectionException(code: 0): An exception occurred in driver: SQLSTATE[HY000] [2002] Connection refused at /Applications/MAMP/htdocs/recettes/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/AbstractMySQLDriver.php:93, Doctrine\\DBAL\\Driver\\PDOException(code: 2002): SQLSTATE[HY000] [2002] Connection refused at /Applications/MAMP/htdocs/recettes/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php:31, PDOException(code: 2002): SQLSTATE[HY000] [2002] Connection refused at /Applications/MAMP/htdocs/recettes/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php:27)","command":"'d:s:u'","message":"An exception occurred in driver: SQLSTATE[HY000] [2002] Connection refused"} []
[2020-02-10 18:39:11] console.DEBUG: Command "'d:s:u'" exited with code "1" {"command":"'d:s:u'","code":1} []
[2020-02-10 18:40:49] console.ERROR: Error thrown while running command "'d:s:u'". Message: "An exception occurred in driver: SQLSTATE[HY000] [2002] Connection refused" {"exception":"[object] (Doctrine\\DBAL\\Exception\\ConnectionException(code: 0): An exception occurred in driver: SQLSTATE[HY000] [2002] Connection refused at /Applications/MAMP/htdocs/recettes/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/AbstractMySQLDriver.php:93, Doctrine\\DBAL\\Driver\\PDOException(code: 2002): SQLSTATE[HY000] [2002] Connection refused at /Applications/MAMP/htdocs/recettes/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php:31, PDOException(code: 2002): SQLSTATE[HY000] [2002] Connection refused at /Applications/MAMP/htdocs/recettes/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php:27)","command":"'d:s:u'","message":"An exception occurred in driver: SQLSTATE[HY000] [2002] Connection refused"} []
[2020-02-10 18:40:49] console.DEBUG: Command "'d:s:u'" exited with code "1" {"command":"'d:s:u'","code":1} []
[2020-02-10 18:41:05] console.ERROR: Error thrown while running command "'d:s:u'". Message: "An exception occurred in driver: SQLSTATE[HY000] [2002] Connection refused" {"exception":"[object] (Doctrine\\DBAL\\Exception\\ConnectionException(code: 0): An exception occurred in driver: SQLSTATE[HY000] [2002] Connection refused at /Applications/MAMP/htdocs/recettes/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/AbstractMySQLDriver.php:93, Doctrine\\DBAL\\Driver\\PDOException(code: 2002): SQLSTATE[HY000] [2002] Connection refused at /Applications/MAMP/htdocs/recettes/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php:31, PDOException(code: 2002): SQLSTATE[HY000] [2002] Connection refused at /Applications/MAMP/htdocs/recettes/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php:27)","command":"'d:s:u'","message":"An exception occurred in driver: SQLSTATE[HY000] [2002] Connection refused"} []
[2020-02-10 18:41:05] console.DEBUG: Command "'d:s:u'" exited with code "1" {"command":"'d:s:u'","code":1} []
[2020-02-10 18:42:33] console.ERROR: Error thrown while running command "'d:s:u'". Message: "An exception occurred in driver: SQLSTATE[HY000] [2002] Connection refused" {"exception":"[object] (Doctrine\\DBAL\\Exception\\ConnectionException(code: 0): An exception occurred in driver: SQLSTATE[HY000] [2002] Connection refused at /Applications/MAMP/htdocs/recettes/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/AbstractMySQLDriver.php:93, Doctrine\\DBAL\\Driver\\PDOException(code: 2002): SQLSTATE[HY000] [2002] Connection refused at /Applications/MAMP/htdocs/recettes/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php:31, PDOException(code: 2002): SQLSTATE[HY000] [2002] Connection refused at /Applications/MAMP/htdocs/recettes/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php:27)","command":"'d:s:u'","message":"An exception occurred in driver: SQLSTATE[HY000] [2002] Connection refused"} []
[2020-02-10 18:42:33] console.DEBUG: Command "'d:s:u'" exited with code "1" {"command":"'d:s:u'","code":1} []
[2020-02-10 18:44:36] console.ERROR: Error thrown while running command "'d:s:u'". Message: "An exception occurred in driver: SQLSTATE[HY000] [2002] Connection refused" {"exception":"[object] (Doctrine\\DBAL\\Exception\\ConnectionException(code: 0): An exception occurred in driver: SQLSTATE[HY000] [2002] Connection refused at /Applications/MAMP/htdocs/recettes/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/AbstractMySQLDriver.php:93, Doctrine\\DBAL\\Driver\\PDOException(code: 2002): SQLSTATE[HY000] [2002] Connection refused at /Applications/MAMP/htdocs/recettes/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php:31, PDOException(code: 2002): SQLSTATE[HY000] [2002] Connection refused at /Applications/MAMP/htdocs/recettes/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php:27)","command":"'d:s:u'","message":"An exception occurred in driver: SQLSTATE[HY000] [2002] Connection refused"} []
[2020-02-10 18:44:36] console.DEBUG: Command "'d:s:u'" exited with code "1" {"command":"'d:s:u'","code":1} []
[2020-02-10 18:45:15] console.ERROR: Error thrown while running command "'d:s:u'". Message: "An exception occurred in driver: SQLSTATE[HY000] [2002] Connection refused" {"exception":"[object] (Doctrine\\DBAL\\Exception\\ConnectionException(code: 0): An exception occurred in driver: SQLSTATE[HY000] [2002] Connection refused at /Applications/MAMP/htdocs/recettes/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/AbstractMySQLDriver.php:93, Doctrine\\DBAL\\Driver\\PDOException(code: 2002): SQLSTATE[HY000] [2002] Connection refused at /Applications/MAMP/htdocs/recettes/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php:31, PDOException(code: 2002): SQLSTATE[HY000] [2002] Connection refused at /Applications/MAMP/htdocs/recettes/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php:27)","command":"'d:s:u'","message":"An exception occurred in driver: SQLSTATE[HY000] [2002] Connection refused"} []
[2020-02-10 18:45:15] console.DEBUG: Command "'d:s:u'" exited with code "1" {"command":"'d:s:u'","code":1} []
[2020-02-10 18:46:06] console.ERROR: Error thrown while running command "'d:s:u'". Message: "An exception occurred in driver: SQLSTATE[HY000] [2002] Connection refused" {"exception":"[object] (Doctrine\\DBAL\\Exception\\ConnectionException(code: 0): An exception occurred in driver: SQLSTATE[HY000] [2002] Connection refused at /Applications/MAMP/htdocs/recettes/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/AbstractMySQLDriver.php:93, Doctrine\\DBAL\\Driver\\PDOException(code: 2002): SQLSTATE[HY000] [2002] Connection refused at /Applications/MAMP/htdocs/recettes/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php:31, PDOException(code: 2002): SQLSTATE[HY000] [2002] Connection refused at /Applications/MAMP/htdocs/recettes/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php:27)","command":"'d:s:u'","message":"An exception occurred in driver: SQLSTATE[HY000] [2002] Connection refused"} []
[2020-02-10 18:46:06] console.DEBUG: Command "'d:s:u'" exited with code "1" {"command":"'d:s:u'","code":1} []
[2020-02-10 18:47:14] console.ERROR: Error thrown while running command "'d:s:u'". Message: "An exception occurred in driver: SQLSTATE[HY000] [2002] No such file or directory" {"exception":"[object] (Doctrine\\DBAL\\Exception\\ConnectionException(code: 0): An exception occurred in driver: SQLSTATE[HY000] [2002] No such file or directory at /Applications/MAMP/htdocs/recettes/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/AbstractMySQLDriver.php:93, Doctrine\\DBAL\\Driver\\PDOException(code: 2002): SQLSTATE[HY000] [2002] No such file or directory at /Applications/MAMP/htdocs/recettes/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php:31, PDOException(code: 2002): SQLSTATE[HY000] [2002] No such file or directory at /Applications/MAMP/htdocs/recettes/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php:27)","command":"'d:s:u'","message":"An exception occurred in driver: SQLSTATE[HY000] [2002] No such file or directory"} []
[2020-02-10 18:47:14] console.DEBUG: Command "'d:s:u'" exited with code "1" {"command":"'d:s:u'","code":1} []
[2020-02-10 18:47:43] console.ERROR: Error thrown while running command "'d:s:u'". Message: "An exception occurred in driver: SQLSTATE[HY000] [1044] Access denied for user 'recettes'@'localhost' to database 'recettes'" {"exception":"[object] (Doctrine\\DBAL\\Exception\\ConnectionException(code: 0): An exception occurred in driver: SQLSTATE[HY000] [1044] Access denied for user 'recettes'@'localhost' to database 'recettes' at /Applications/MAMP/htdocs/recettes/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/AbstractMySQLDriver.php:93, Doctrine\\DBAL\\Driver\\PDOException(code: 1044): SQLSTATE[HY000] [1044] Access denied for user 'recettes'@'localhost' to database 'recettes' at /Applications/MAMP/htdocs/recettes/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php:31, PDOException(code: 1044): SQLSTATE[HY000] [1044] Access denied for user 'recettes'@'localhost' to database 'recettes' at /Applications/MAMP/htdocs/recettes/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php:27)","command":"'d:s:u'","message":"An exception occurred in driver: SQLSTATE[HY000] [1044] Access denied for user 'recettes'@'localhost' to database 'recettes'"} []
[2020-02-10 18:47:43] console.DEBUG: Command "'d:s:u'" exited with code "1" {"command":"'d:s:u'","code":1} []
[2020-02-10 18:48:40] doctrine.DEBUG: SHOW FULL TABLES WHERE Table_type = 'BASE TABLE' [] []
[2020-02-10 18:48:41] console.DEBUG: Command "'d:s:u'" exited with code "1" {"command":"'d:s:u'","code":1} []
[2020-02-10 18:48:49] doctrine.DEBUG: SHOW FULL TABLES WHERE Table_type = 'BASE TABLE' [] []
[2020-02-10 18:48:49] doctrine.DEBUG: SHOW FULL TABLES WHERE Table_type = 'BASE TABLE' [] []
[2020-02-10 18:48:49] doctrine.DEBUG: CREATE TABLE user (id INT AUTO_INCREMENT NOT NULL, email VARCHAR(180) NOT NULL, roles JSON NOT NULL, password VARCHAR(255) NOT NULL, firstname VARCHAR(255) DEFAULT NULL, UNIQUE INDEX UNIQ_8D93D649E7927C74 (email), PRIMARY KEY(id)) DEFAULT CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci ENGINE = InnoDB [] []
[2020-02-10 18:50:38] doctrine.DEBUG: SHOW FULL TABLES WHERE Table_type = 'BASE TABLE' [] []
[2020-02-10 18:50:38] doctrine.DEBUG: SELECT COLUMN_NAME AS Field, COLUMN_TYPE AS Type, IS_NULLABLE AS `Null`, COLUMN_KEY AS `Key`, COLUMN_DEFAULT AS `Default`, EXTRA AS Extra, COLUMN_COMMENT AS Comment, CHARACTER_SET_NAME AS CharacterSet, COLLATION_NAME AS Collation FROM information_schema.COLUMNS WHERE TABLE_SCHEMA = 'recettes' AND TABLE_NAME = 'user' ORDER BY ORDINAL_POSITION ASC [] []
[2020-02-10 18:50:38] doctrine.DEBUG: SELECT DISTINCT k.`CONSTRAINT_NAME`, k.`COLUMN_NAME`, k.`REFERENCED_TABLE_NAME`, k.`REFERENCED_COLUMN_NAME` /*!50116 , c.update_rule, c.delete_rule */ FROM information_schema.key_column_usage k /*!50116 INNER JOIN information_schema.referential_constraints c ON   c.constraint_name = k.constraint_name AND   c.table_name = 'user' */ WHERE k.table_name = 'user' AND k.table_schema = 'recettes' /*!50116 AND c.constraint_schema = 'recettes' */ AND k.`REFERENCED_COLUMN_NAME` is not NULL [] []
[2020-02-10 18:50:38] doctrine.DEBUG: SELECT TABLE_NAME AS `Table`, NON_UNIQUE AS Non_Unique, INDEX_NAME AS Key_name, SEQ_IN_INDEX AS Seq_in_index, COLUMN_NAME AS Column_Name, COLLATION AS Collation, CARDINALITY AS Cardinality, SUB_PART AS Sub_Part, PACKED AS Packed, NULLABLE AS `Null`, INDEX_TYPE AS Index_Type, COMMENT AS Comment FROM information_schema.STATISTICS WHERE TABLE_NAME = 'user' AND TABLE_SCHEMA = 'recettes' ORDER BY SEQ_IN_INDEX ASC [] []
[2020-02-10 18:50:38] doctrine.DEBUG: SELECT ENGINE, AUTO_INCREMENT, TABLE_COLLATION, TABLE_COMMENT, CREATE_OPTIONS FROM information_schema.TABLES WHERE TABLE_TYPE = 'BASE TABLE' AND TABLE_SCHEMA = DATABASE() AND TABLE_NAME = 'user' [] []
[2020-02-10 18:50:38] doctrine.DEBUG: SHOW FULL TABLES WHERE Table_type = 'BASE TABLE' [] []
[2020-02-10 18:50:38] doctrine.DEBUG: SELECT COLUMN_NAME AS Field, COLUMN_TYPE AS Type, IS_NULLABLE AS `Null`, COLUMN_KEY AS `Key`, COLUMN_DEFAULT AS `Default`, EXTRA AS Extra, COLUMN_COMMENT AS Comment, CHARACTER_SET_NAME AS CharacterSet, COLLATION_NAME AS Collation FROM information_schema.COLUMNS WHERE TABLE_SCHEMA = 'recettes' AND TABLE_NAME = 'user' ORDER BY ORDINAL_POSITION ASC [] []
[2020-02-10 18:50:38] doctrine.DEBUG: SELECT DISTINCT k.`CONSTRAINT_NAME`, k.`COLUMN_NAME`, k.`REFERENCED_TABLE_NAME`, k.`REFERENCED_COLUMN_NAME` /*!50116 , c.update_rule, c.delete_rule */ FROM information_schema.key_column_usage k /*!50116 INNER JOIN information_schema.referential_constraints c ON   c.constraint_name = k.constraint_name AND   c.table_name = 'user' */ WHERE k.table_name = 'user' AND k.table_schema = 'recettes' /*!50116 AND c.constraint_schema = 'recettes' */ AND k.`REFERENCED_COLUMN_NAME` is not NULL [] []
[2020-02-10 18:50:38] doctrine.DEBUG: SELECT TABLE_NAME AS `Table`, NON_UNIQUE AS Non_Unique, INDEX_NAME AS Key_name, SEQ_IN_INDEX AS Seq_in_index, COLUMN_NAME AS Column_Name, COLLATION AS Collation, CARDINALITY AS Cardinality, SUB_PART AS Sub_Part, PACKED AS Packed, NULLABLE AS `Null`, INDEX_TYPE AS Index_Type, COMMENT AS Comment FROM information_schema.STATISTICS WHERE TABLE_NAME = 'user' AND TABLE_SCHEMA = 'recettes' ORDER BY SEQ_IN_INDEX ASC [] []
[2020-02-10 18:50:38] doctrine.DEBUG: SELECT ENGINE, AUTO_INCREMENT, TABLE_COLLATION, TABLE_COMMENT, CREATE_OPTIONS FROM information_schema.TABLES WHERE TABLE_TYPE = 'BASE TABLE' AND TABLE_SCHEMA = DATABASE() AND TABLE_NAME = 'user' [] []
[2020-02-10 18:50:38] doctrine.DEBUG: ALTER TABLE user ADD lastname VARCHAR(255) DEFAULT NULL, ADD username VARCHAR(255) NOT NULL [] []
[2020-02-10 18:53:51] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /" (from "http://localhost:8888/recettes/")" at /Applications/MAMP/htdocs/recettes/vendor/symfony/http-kernel/EventListener/RouterListener.php line 136 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /\" (from \"http://localhost:8888/recettes/\") at /Applications/MAMP/htdocs/recettes/vendor/symfony/http-kernel/EventListener/RouterListener.php:136, Symfony\\Component\\Routing\\Exception\\NoConfigurationException(code: 0):  at /Applications/MAMP/htdocs/recettes/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:182)"} []
